composer.json

```diff
@@ -14,13 +14,15 @@
         "illuminate/support": "^5.4 || ^6.0 || ^7.0"
     },
     "require-dev": {
-        "laravel/lumen-framework": "^5.4 || ^6.0 || ^7.0",
-        "codeception/assert-throws": "^1.0",
-        "codeception/codeception": "^2.5",
+        "codeception/assert-throws": "^1.1",
+        "codeception/codeception": "^4.1",
+        "codeception/module-asserts": "^1.3",
         "codeception/specify": "^1.2",
         "codeception/verify": "^1.2",
+        "laravel/lumen-framework": "^5.4 || ^6.0 || ^7.0",
         "php-coveralls/php-coveralls": "^2.2",
-        "phpstan/phpstan": "^0.12.2"
+        "phpstan/phpstan": "^0.12.2",
+        "phpstan/phpstan-symfony": "^0.12.11"
```

Comment:
This seems to be a new Composer default (to specify the `z` in `^x.y.z`). I went with it, let me know if you want me to change it to `^x.y` like the rest.